client/ayon_blender/api/ops.py

```diff
@@ -188,7 +189,22 @@ def _process_app_events() -> Optional[float]:
                 msg)
             dialog.setMinimumWidth(500)
             dialog.setDetailedText(detail)
-            dialog.exec_()
+            dialog.setAttribute(QtCore.Qt.WA_DeleteOnClose)
+            dialog.setStyleSheet(load_stylesheet())
+            # Ensure the dialog stays on top and is properly focused
+            dialog.setWindowFlags(
+                dialog.windowFlags() |
+                QtCore.Qt.WindowStaysOnTopHint |
+                QtCore.Qt.Dialog
+            )
+
+            try:
```

Comment:
Why this is in try block with `deleteLater`?
1. Not sure what could fail?
2. If it fails it is cleaned up from memory and python+shiboken will take care of it.

Reply:
If shiboken would take care of it, I would remove this then.

Reply:
Also with using `open`, you don't want to call `deleteLater()` as it would destroy the window before it is closed 🙂.